token-client/README.md

```diff
@@ -168,12 +162,37 @@ To consume the `XsuaaTokenFlows` class, you simply need to `@Autowire` it like t
 private XsuaaTokenFlows xsuaaTokenFlows;
 ```
 
-For X.509 based authentication method using an externally managed certificate, `ClientCertificate` class needs to be instantiated with the external key. For `spring-xsuaa` based applications it can be easily done by overriding these values in `application.yml properties file.
-```yaml
-# For externally managed X.509 certificate
-xsuaa:
-  key: -----BEGIN RSA PRIVATE KEY-----YOUR PRIVATE KEY-----END RSA PRIVATE KEY-----
-```
+For X.509 based authentication method using an externally managed certificate, `ClientCertificate` class needs to be instantiated with the external key and `xsuaaTokenFlows` bean needs to be overwritten using this `ClientCertificate` instance. <br>Alternatively you can provide the certificate key property as command-line argument or programmatically providing default property.
```

Comment:
@nenaraab then we don't mention command line argument example? do you think it's not relevant?

Reply:
exactly. i see no benefit for productive usage :-)